Makefile

```diff
@@ -153,7 +153,7 @@ clean-labels:
 # Generate code
 generate: controller-gen mockgen
 	$(CONTROLLER_GEN) object:headerFile="utils/boilerplate.go.txt" paths="./..."
-	$(GO_CMD) generate ./...
+	go generate ./...
```

Comment:
we should probably either use $(go_CMD) everywhere, or hardcode "go" everywhere rather than the mix we have at the moment, but that's a problem for another day :)

Reply:
I will fix it in a different PR